src/main/resources/config.yml

```diff
@@ -246,6 +246,10 @@ old-fishing-knockback:
   # This is the delay in milliseconds in-between rod damage, so the player hit has time to fall back down
   hitCooldown: 1000
 
+old-fishing-rod-velocity:
+  # This is to revert the fishing rod cast to pre-1.9 velocity
```

Comment:
Please provide a little bit more context, such as that the fishing rod goes farther etc.